src/components/CancerHallmarks/Detail.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import Typography from '@material-ui/core/Typography';
+
+import { Link, OtTableRF, DataDownloader } from 'ot-ui';
+
+const query = gql`
+  query CancerHallmarksQuery($ensgId: String!) {
+    target(ensgId: $ensgId) {
+      id
+      details {
+        cancerHallmarks {
+          publicationsByHallmark {
+            name
+            promotes
+            suppresses
+            publications {
+              pmId
+              description
+            }
+          }
+          roleInCancer {
+            name
+          }
+        }
+      }
+    }
+  }
+`;
+
+const columns = [
+  {
+    id: 'hallmark',
+    label: 'Hallmark',
+  },
+  {
+    id: 'sources',
+    label: 'Sources',
+  },
+  {
+    id: 'note',
+    label: 'Notes',
+  },
+];
+
+const CancerHallmarksDetail = ({ ensgId, symbol, sources }) => {
+  return (
+    <React.Fragment>
```

Comment:
`Query` is the only child here, so `Fragment` is not needed in this situation.

Reply:
Thank you Alfredo. Original PR was really just for the overview widget. I'm finishing the detail view and will update the PR shortly